src/draggable.ts

```diff
@@ -5,10 +5,16 @@
 
 
 import {EventTarget, applyDefaults} from '@mathigon/core';
-import {Bounds, Point} from '@mathigon/euclid';
+import {Point} from '@mathigon/euclid';
 import {SVGParentView, ElementView, $html} from './elements';
 import {Browser} from './browser';
 import {slide} from './events';
+import {animate} from './animate';
+
+
+type HoverData =
+  {tag: 'Target', $el: ElementView} |
+  {tag: 'NonTarget'};
```

Comment:
Not sure why we need this. Could we not just have `over: ElementView|undefined`?